File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/utente/utente.ts
import { Utente_Impostazioni } from "./utente_impostazioni";
import { Utente_Permesso } from "./utente_permesso";
import { Messaggio_Utente_Permessi } from "./messaggio_utente_permessi";

export class Utente {
    constructor (
        public Username?: string,
        public Nome?: string,
        public Cognome?: string,
        public Rag_Soc?: string,
        public Cod_Fisc?: string,
        public UsernameCommerciale?: string,
        public Permessi: Utente_Permesso[] = [],
        public Impostazioni: Utente_Impostazioni[] = [],
        public ValiditaUtentePermessiLicenza?: Messaggio_Utente_Permessi
    ) {}
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/utente/Utente.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.utente
{
    public class Utente
    {
        public String Username { get; set; }
        public String Nome { get; set; }
        public String Cognome { get; set; }
        public String Rag_Soc { get; set; }
        public String Cod_Fisc { get; set; }
        public String UsernameCommerciale { get; set; }
        public List<Utente_Permesso> Permessi { get; set; }
        public List<Utente_Impostazioni> Impostazioni { get; set; }
        public Messaggio_Utente_Permessi ValiditaUtentePermessiLicenza { get; set; }
    }
}
